propulsion-reactor/Program.fs

```diff
@@ -357,7 +357,7 @@ module Args =
             match tcp, port with
             | false, None ->   Discovery.GossipDns            host
             | false, Some p -> Discovery.GossipDnsCustomPort (host, p)
-            | true, None ->    Discovery.Uri                 (UriBuilder("tcp", host).Uri)
+            | true, None ->    Discovery.Uri                 (UriBuilder("tcp", host, 1113).Uri)
```

Comment:
there's a clone of this in `propulsion-sync` too

Reply:
Done